### PR TITLE
Rewriting cloud manage auth

### PR DESCRIPTION
**Pull request by frankeros**
## Description

Hi team, this PR update the section Authentication and authorization of the Cloud docu
separating the current guide into different topics:
- creating an internal user and mapping to Wazuh
- given admin permissions to an internal user and mapping as well to Wazuh
- given read-only permissions to an internal user and mapping as well to Wazuh

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

### CODE REVIEW

**Review by Robin350 — APPROVED** [review]
LGTM ✅

**Review by MarVergara — APPROVED** [review]
LGTM

File: source/cloud-service/your-environment/manage-auth.rst [hunk 1]
```diff
@@ -18,50 +18,121 @@ You can use the native support for managing and authenticating users, or integra
...
+- `Creating and setting a Wazuh read-only user`_
 
-1. :ref:`Log into your WUI<cloud_getting_started_wui_access>` as administrator.
 
-2. Click the upper-left menu icon to open the options, select **Security** and then **Internal users** to open the internal users page.
+Creating an internal user and mapping it to Wazuh
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-3. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to create a user.
+Follow these steps to create an internal user, map it to its appropriate role, and create a new role mapping.
```
Comment (s-ocando) [inline source/cloud-service/your-environment/manage-auth.rst:31]:
```suggestion
Follow these steps to create an internal user and map it to its appropriate role.
```

File: source/cloud-service/your-environment/manage-auth.rst [hunk 2]
```diff
@@ -18,50 +18,121 @@ You can use the native support for managing and authenticating users, or integra
...
 
-4. Go to **Security** and select **Roles** to open the page.
+#. :ref:`Log into your WUI<cloud_getting_started_wui_access>` as administrator.
 
-5. Click **Create role**, complete the empty fields, and then click **Create** to complete the task. 
+#. Click the upper-left menu icon to open the options, select **Security** and then **Internal users** to open the internal users page.
+
+#. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to create a user.
```
Comment (s-ocando) [inline source/cloud-service/your-environment/manage-auth.rst:37]:
```suggestion
#. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to complete the action.
```

File: source/cloud-service/your-environment/manage-auth.rst [hunk 3]
```diff
@@ -18,50 +18,121 @@ You can use the native support for managing and authenticating users, or integra
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   - Tenant permissions: ``global_tenant`` and select the **Read only** option.
+Follow these steps to create an internal user, create a new role mapping, and give administrator permissions to the user.
+
+#. :ref:`Log into your WUI<cloud_getting_started_wui_access>` as administrator.
+
+#. Click the upper-left menu icon to open the options, select **Security** and then **Internal users** to open the internal users page.
+
+#. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to create a user.
```
Comment (s-ocando) [inline source/cloud-service/your-environment/manage-auth.rst:66]:
```suggestion
#. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to complete the action.
```

File: source/cloud-service/your-environment/manage-auth.rst [hunk 4]
```diff
@@ -18,50 +18,121 @@ You can use the native support for managing and authenticating users, or integra
...
+Creating and setting a Wazuh read-only user
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Follow these steps to create an internal user, create a new role mapping, and give read-only permissions to the user.
+
+#. :ref:`Log into your WUI<cloud_getting_started_wui_access>` as administrator.
+
+#. Click the upper-left menu icon to open the options, select **Security** and then **Internal users** to open the internal users page.
+
+#. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to create a user.
```
Comment (s-ocando) [inline source/cloud-service/your-environment/manage-auth.rst:100]:
```suggestion
#. Click **Create internal user**, complete the empty fields with the requested information, and click **Create** to complete the action.
```

File: source/cloud-service/your-environment/manage-auth.rst [hunk 5]
```diff
@@ -18,50 +18,121 @@ You can use the native support for managing and authenticating users, or integra
...
-   - Name: Assign a name to the role.
-     
-   - Cluster permissions: ``cluster_composite_ops_ro``
+#. To map the user with Wazuh, follow these steps:
+   
+   #. Go to the Wazuh WUI, click **Wazuh** to open the menu, select **Security**, and then **Roles mapping** to open the page.
+   #. Click **Create Role mapping** and complete the empty fields with the following parameters:
+   
+      - **Role mapping name**: Assign a name to the role mapping.
+      - **Roles**: Select the Wazuh roles that you want to map with the user.
```
Comment (s-ocando) [inline source/cloud-service/your-environment/manage-auth.rst:51]:
```suggestion
      - **Roles**: Select the Wazuh roles that you want to map the user with.
```

**Review by MarVergara — APPROVED** [review]
LGTM

**Review by s-ocando — APPROVED** [review]

**Pull request merged**